homeassistant/components/smartthings/switch.py

```diff
@@ -29,44 +31,104 @@
 )
 
 
+@dataclass(frozen=True, kw_only=True)
+class SmartThingsSwitchEntityDescription(SwitchEntityDescription):
+    """Describe a SmartThings switch entity."""
+
+    status_attribute: Attribute
+    exists_fn: Callable[[dict[Attribute | str, Status]], bool] = lambda _: True
+
+
+SWITCH = SmartThingsSwitchEntityDescription(
+    key=Capability.SWITCH,
+    status_attribute=Attribute.SWITCH,
+    name=None,
+)
+CAPABILITY_TO_SWITCHES: dict[Capability | str, SmartThingsSwitchEntityDescription] = {
+    Capability.SAMSUNG_CE_AIR_CONDITIONER_LIGHTING: SmartThingsSwitchEntityDescription(
+        key=Capability.SAMSUNG_CE_AIR_CONDITIONER_LIGHTING,
```

Comment:
Is it a light or a switch? The name sounds like it's a light, but you're adding it as a switch entity.

Reply:
Oh it controls the display, I should probably fix the naming of this entity